setup.py

```diff
@@ -60,7 +60,7 @@ def _load_py_module(fname, pkg="flash"):
     url=about.__homepage__,
     download_url="https://github.com/PyTorchLightning/lightning-flash",
     license=about.__license__,
-    packages=find_packages(exclude=["tests", "docs"]),
```

Comment:
`"docs"` isn't necessary here because it's not included in distribution anyway as it's not a package.